simulator/src/main/java/com/clearspring/analytics/stream/frequency/CountMinTinyLFU.java

```diff
@@ -0,0 +1,56 @@
+package com.clearspring.analytics.stream.frequency;
```

Comment:
This doesn't appear to need to be in their library's package. Instead put it in our admission package as package private.